diff -up plasma-vault-6.3.3/kded/engine/commandresult.h.omv~ plasma-vault-6.3.3/kded/engine/commandresult.h
--- plasma-vault-6.3.3/kded/engine/commandresult.h.omv~	2025-03-20 18:46:49.316596329 +0100
+++ plasma-vault-6.3.3/kded/engine/commandresult.h	2025-03-20 18:46:59.733673191 +0100
@@ -8,6 +8,7 @@
 #define PLASMAVAULT_KDED_ENGINE_COMMAND_RESULT_H
 
 #include <QFuture>
+#include <QDebug>
 
 #include "asynqt/basic/all.h"
 #include "asynqt/utils/expected.h"
